RFCs/0009-Editors-API-RevisionUpdate.md

```diff
@@ -0,0 +1,350 @@
+      "ElapsedTimeInRunningTests": "00:00:00.8677523"
+    },
+    "LastRunTests": {
+      "NewTestResults": [
+        {
+          "TestCase": {
+            "Id": "850ad69f-0dc9-fb92-8500-8d2f8d8dfe2a",
+            "FullyQualifiedName": "_20TestProject.UnitTest1.Test1",
+            "DisplayName": "Test1",
+            "ExecutorUri": "executor://MSTestAdapter/v2",
+            "Source": "C:\\\\Users\\\\sasin\\\\Documents\\\\Visual Studio 2017\\\\Projects\\\\20TestProject\\\\20TestProject\\\\bin\\\\Debug\\\\net452\\\\20TestProject.dll",
+            "CodeFilePath": null,
+            "LineNumber": 0,
+            "Properties": [
+              {
+                "Key": {
+                  "Id": "MSTestDiscovererv2.IsEnabled",
+                  "Label": "IsEnabled",
+                  "Category": "",
+                  "Description": "",
+                  "Attributes": 1,
+                  "ValueType": "System.Boolean"
+                },
+                "Value": true
+              },
+              {
+                "Key": {
+                  "Id": "MSTestDiscovererv2.TestClassName",
+                  "Label": "ClassName",
+                  "Category": "",
+                  "Description": "",
+                  "Attributes": 1,
+                  "ValueType": "System.String"
+                },
+                "Value": "_20TestProject.UnitTest1"
+              },
+              {
+                "Key": {
+                  "Id": "TestObject.Traits",
+                  "Label": "Traits",
+                  "Category": "",
+                  "Description": "",
+                  "Attributes": 5,
+                  "ValueType": "System.Collections.Generic.KeyValuePair`2[[System.String],[System.String]][]"
+                },
+                "Value": []
+              }
+            ]
+          },
+          "Attachments": [],
+          "Outcome": 1,
+          "ErrorMessage": null,
+          "ErrorStackTrace": null,
+          "DisplayName": null,
+          "Messages": [],
+          "ComputerName": null,
+          "Duration": "00:00:00.0072901",
+          "StartTime": "2017-03-20T19:57:18.2262042+05:30",
+          "EndTime": "2017-03-20T19:57:18.2921987+05:30",
+          "Properties": []
+        }
+      ],
+      "TestRunStatistics": {
+        "ExecutedTests": 1,
+        "Stats": {
+          "Passed": 1
+        }
+      },
+      "ActiveTests": []
+    },
+    "RunAttachments": [],
+    "ExecutorUris": [
+      "executor://mstestadapter/v2"
+    ]
+  }
+}
+```
+
+All the other messages like Cancel, Abort or debugging related messages follow the same pattern.
+
+## Error message for ProtocolVersion mismatch
+
+##### API Payload
+| Key         | Type   | Description                                  |
+|-------------|--------|----------------------------------------------|
+| MessageType | string | ProtocolError                                |
+| Payload     | object | String containing the supported range        |
```

Comment:
Should we support these:
1. Where did the version mismatch occur?
2. What's the range supported?